manager/api/tests/test_jira.py

```diff
@@ -311,6 +316,27 @@ def test_needed_parameters(self):
 
         mock_jira_patcher.stop()
 
+    @patch("requests.get")
+    @patch("requests.put")
+    def test_update_time_loss(self, mock_get, mock_put):
```

Comment:
Perhaps you could also test when the `update_time_loss` fails and returns a 400 status code, ensuring we return the arranged `ack` (e.g. "Jira time_lost field could not be updated" in case you accept my other suggestion too).

Reply:
I learned that when we use requests.Response() as our patch in the unit tests, that this is not the same as rest_framework.response.Response (that we set in utils.py). So when we start to mock one of these, we cannot then check what the 'ack' would have been, since we are patching that object.